packs/bestiary-effects/effect-grab-debris.json

```diff
@@ -0,0 +1,124 @@
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 5
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Howl of the Wild"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "self:effect:grab-debris:1"
+                ],
+                "selector": "ac",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "attackModifier": 12,
+                "damage": {
+                    "base": {
+                        "damageType": "slashing",
+                        "dice": 2,
+                        "die": "d10",
+                        "modifier": 6
+                    }
+                },
+                "key": "Strike",
+                "label": "PF2E.NPCAbility.Scroungefeather.GrabDebris.BrokenBlade",
+                "predicate": [
+                    "self:effect:grab-debris:2"
+                ],
+                "slug": "broken-blade",
+                "traits": []
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    "item:slug:tossed-scraps",
+                    "self:effect:grab-debris:3"
+                ],
+                "property": "damage-type",
+                "selectors": [
+                    "inline-damage"
+                ],
+                "value": "fire"
+            },
+            {
+                "damageCategory": "persistent",
+                "damageType": "fire",
+                "key": "FlatModifier",
+                "predicate": [
+                    "self:effect:grab-debris:3"
+                ],
+                "selector": "tossed-scraps-inline-damage",
+                "value": 3
+            },
+            {
+                "attackModifier": 14,
+                "damage": {
+                    "base": {
+                        "damageType": "force",
+                        "dice": 2,
+                        "die": "d6",
+                        "modifier": 6
+                    }
+                },
+                "key": "Strike",
+                "label": "PF2E.NPCAbility.Scroungefeather.GrabDebris.MagicBolt",
+                "predicate": [
+                    "self:effect:grab-debris:4"
+                ],
+                "range": {
+                    "max": 30
```

Comment:
No range is listed, so I arbitrarily chose 30.

Reply:
It's 20 feet, as per the tossed scraps action